tests/unit_tests/test_flight_boundaries_container.py

```diff
@@ -0,0 +1,12 @@
+from SDA import *
+import numpy as np
+from SUASSystem import *
+import unittest
+
+class FlightBoundaryContainerTestCase(unittest.TestCase):
+
+    def setUp(self):
+        self.boundary = FlightBoundariesContainer(np.array([np.array([[-2000, -2000], [-2000, 2000], [2000, 2000], [2000, -2000]])]))
```

Comment:
The variable name should be more descriptive. Since there is a boundary object, you should instead call this ```self.test_boundary_container```

Reply:
done